data/json/overmap/map_extras.json

```
@@ -387,5 +387,15 @@
     "name": "Prison Bus",
     "description": "A prison bus.",
     "generator": { "generator_method": "update_mapgen", "generator_id": "mx_prison_bus" }
+  },
+  {
+    "id": "mx_mass_grave",
+    "type": "map_extra",
+    "name": "Mass Grave",
+    "description": "A mass grave.",
+    "generator": { "generator_method": "update_mapgen", "generator_id": "mx_mass_grave" },
+    "sym": "x",
```

To match the previous overmap tile.
```suggestion
    "sym": "X",
```